spring-boot-project/spring-boot/src/main/java/org/springframework/boot/web/embedded/tomcat/TomcatReactiveWebServerFactory.java

```diff
@@ -60,11 +60,11 @@
 	/**
 	 * The class name of default protocol used.
 	 */
-	public static final String DEFAULT_PROTOCOL = "org.apache.coyote.http11.Http11NioProtocol";
+	private static final String DEFAULT_PROTOCOL = "org.apache.coyote.http11.Http11NioProtocol";
```

Comment:
I don't think we can make this `private` in case someone else is relying on it.

Reply:
You're right, sorry. I was thinking like someone working on their own final codebase as opposed to one that's used by others as a library :)